pkg/customflag/model.go

```diff
@@ -9,73 +9,73 @@ import (
 )
 
 var ServiceFlag FlagConfig
-var TestCaseNameFlag StringSlice
+var TestCaseNameFlag stringSlice
 
 // FlagConfig is a type that wraps all the flags that can be used
 type FlagConfig struct {
-	InstallMode       InstallTypeValueFlag
-	TestConfig        TestConfigFlag
-	ClusterConfig     ClusterConfigFlag
-	SUCUpgradeVersion SUCUpgradeVersion
-	Channel           ChannelFlag
+	InstallMode       installTypeValueFlag
```

Comment:
Can we rename it to be consistent? `installModeFlag` instead of `installTypeValueFlag`?

Reply:
totally